playbook.yml

```diff
@@ -0,0 +1,8 @@
+- hosts: all
+
+  tasks: 
+    - shell: echo hello world
```

Comment:
Commands should either read information (and thus set ``changed_when``) or not do something if it has already been done (using creates/removes) or only do it if another check has a particular result (``when``)